src/components/Preloader/Preloader.module.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../styles/variables';
+
+.progress {
+  position: relative;
+  display: block;
+  overflow: hidden;
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  margin: 1rem 0;
+  background-color: $preloader-bg;
+}
+
+.progress .indeterminate {
+  background-color: $main-color;
+}
+
+.progress .indeterminate::before {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  animation: indeterminate 2.1s cubic-bezier(0.65, 0.815, 0.735, 0.395) infinite;
+  background-color: inherit;
+  content: '';
+  will-change: left, right;
+}
+
+.progress .indeterminate::after {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  animation: indeterminate-short 2.1s cubic-bezier(0.165, 0.84, 0.44, 1) infinite;
+  animation-delay: 1.15s;
+  background-color: inherit;
+  content: '';
+  will-change: left, right
+}
+
+@keyframes indeterminate {
+  0% {
+    right: 100%;
+    left: -35%;
+  }
+
+  60% {
+    right: -90%;
+    left: 100%;
+  }
+
+  100% {
+    right: -90%;
+    left: 100%;
+  }
+}
```

Comment:
check empty spaces) and why u delete stylelint?

Reply:
add spaces